### ISSUE TITLE
build fails with ADL_SDK v8

### ISSUE THREAD

**Issue by pszi1ard** [epixoip/od6config#4]
With the latest ADL SDK I get the following error:

```
cc -g -O2 -Wall    -c -o src/adapter_info.o src/adapter_info.c
In file included from src/../ADL_SDK/include/adl_sdk.h:19:0,
                 from src/od6config.h:14,
                 from src/adapter_info.c:15:
src/../ADL_SDK/include/adl_structures.h:2346:5: error: unknown type name ‘bool’
     bool  countValid;
     ^
src/../ADL_SDK/include/adl_structures.h:2352:5: error: unknown type name ‘bool’
     bool locationValid;
     ^
make: *** [src/adapter_info.o] Error 1
```

**Comment by epixoip** [comment]
This is a bug in ADL SDK 8.0. They rely on C99 but fail to include stdbool.h, and/or define their own type for 'bool'. We can work around this though by defining it ourselves in od6config.h

**Comment by epixoip** [comment]
And actually I don't know if you noticed but as of commit 2c13d2ff28 od6config includes ADL SDK 6.0 so there's no need to download it and copy it into the directory anymore. My fault for not updating the Readme. However, I will take this opportunity to upgrade it to 8.0

**Comment by epixoip** [comment]
Fixed in commit df5198b and commit e323139

**Comment by pszi1ard** [comment]
I did notice that there were some file in the source tree, but I did not realize that it was the full ADL SDK.

Thanks for the fixes!
